iam/google/cloud/iam/credentials.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from google.cloud.iam.credentials_v1 import IAMCredentialsClient
+from google.cloud.iam.credentials_v1 import types
+
+__all__ = ("types", "IAMCredentialsClient")
```

Comment:
This directory structure does not fit our normal pattern, which would be something like `google/cloud/iam_credentials_v1/` for the service.  Since `google.cloud.iam` is not a namespace package, the extra layer provides no benefit.

Reply:
While I can't be sure I assume there is a reason? I will look into the configuration yaml for the generation and see if this was an error.